docs/en/observability/apm/known-issues.asciidoc

```diff
@@ -19,6 +19,83 @@ If applicable, exact error messages linked to this issue so users searching for
 Link to fix
 ////
 
+*Upgrading to v8.15.0 may cause APM indices to lose their lifecycle policy* +
+_Elastic Stack versions: 8.15.0_ +
+_Fixed in Elastic Stack version 8.15.1_
+
+// The conditions in which this issue occurs
+The issue only occurs when _upgrading_ the {stack} to 8.15.0.
+The issue does _not_ occur when creating a _new_ cluster using 8.15.0.
+The issue also does not occur if a custom ILM policy is configured using a custom component template.
+
+// Describe why it happens
+This happens because APM Server now uses the Elasticsearch apm-data plugin to manage
+index templates, and as a result APM Server now uses datastream lifecycle (DSL) instead
+of index lifecycle management (ILM) to manage data retention for APM indices.
+This means that any datastream created before 8.15.0 does not have DSL configuration
+and, when upgraded to 8.15.0, the datastream becomes unmanaged because it has
+no DSL configuration. 
```

Comment:
Is it important to call out the switch to apm-data plugin? I wonder what the user benefit for this level of detail is, and whether we should simply focus on the switch to DSL; something like:

> in 8.15.0, APM Server switched to use datastream lifecycle (DSL) to manage data retention for APM indices for new deployments as well as for upgraded deployments with default lifecycle configurations. Unfortunately, since any datastream created before `8.15.0` does not have DSL configuration, such existing datastreams become unmanaged.

Reply:
Sounds good! Updated!